src/main/java/com/romraider/logger/external/innovate/lm2/mts/plugin/Lm2MtsDataSource.java

```diff
@@ -59,18 +62,21 @@
      * one stream can be processed.
      */
     {
-        final MTSConnector connector = new MTSConnector();
-        int[] ports = connector.getMtsPorts();
-        if (ports != null) {
-            for (int i = 0; i < ports.length; i++) {
-                connector.usePort(i);
-                dataItems = connector.getSensors();
-            }
-        }
-        else {
-            throw new IllegalStateException("Innovate LogWorks MTS control does not appear to be installed on this computer");
-        }
-        connector.dispose();
+    	try {
+	        final MTSConnector connector = new MTSConnector();
+	        int[] ports = connector.getMtsPorts();
+	        if (ports != null) {
+	            for (int i = 0; i < ports.length; i++) {
+	                connector.usePort(i);
+	                dataItems = connector.getSensors();
+	            }
+	        }
+	        
+	        connector.dispose();
+    	}
+    	catch(com4j.ExecutionException e) {
+    		throw new PluginNotInstalledException("Innovate LogWorks MTS control does not appear to be installed on this computer");
```

Comment:
Will this message be displayed to the user?  If so it needs to go into a locale properties file.

Reply:
It will be inside the log, but there will not be a window popup, neither will it be seen anywhere in the gui